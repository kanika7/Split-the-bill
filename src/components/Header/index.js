import React from "react";
import { PAGE_TITLE } from "../../constants";
import "./styles/index.scss";

const Header = () => (
  <header className="navbar-fixed-top">
    <div className="nav-container">
      <header>{PAGE_TITLE}</header>
    </div>
  </header>
);
export default Header;
