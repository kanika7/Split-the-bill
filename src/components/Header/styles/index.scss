@import "./default.scss";
@import "./tablet.scss";
@import "./desktop.scss";
