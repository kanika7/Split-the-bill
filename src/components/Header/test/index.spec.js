import React from "react";
import { shallow } from "enzyme";
import Header from "../index";

describe("<Header /> Test Suits", () => {
  let wrapper;
  function renderShallow() {
    wrapper = shallow(<Header />);
  }

  it("renders snapshot", () => {
    renderShallow();
    expect(wrapper).toMatchSnapshot();
  });
});
