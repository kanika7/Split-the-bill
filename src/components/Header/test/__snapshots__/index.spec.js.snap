// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Test Suits renders snapshot 1`] = `
<header
  className="navbar-fixed-top"
>
  <div
    className="nav-container"
  >
    <header>
      Split The Bill
    </header>
  </div>
</header>
`;
